_includes/header.html

```diff
@@ -11,7 +11,11 @@
       <a class="nav-link" href="sign_up_form.html">Sign up form</a>
     </li>
     <li class="nav-item">
+<<<<<<< HEAD
```

Comment:
I am going to revert the merge of the Pull Request as the unresolved merge conflicts will cause the site to break.